lib/seattleflu/id3c/cli/command/etl/redcap_det_scan.py

```diff
@@ -534,7 +580,11 @@ def create_resource_condition(record: dict, symptom_name: str, patient_reference
 
 def create_specimen(record: dict, patient_reference: dict) -> tuple:
     """ Returns a FHIR Specimen resource entry and reference """
-    barcode = record['return_utm_barcode']
+    # SCAN In-Person Enrollments project does not have the `return_utm_barcode`
+    # field, so use `utm_tube_barcode_2` instead.
+    # -Jover, 16 July 2020
+    barcode = record.get('return_utm_barcode') or record.get('utm_tube_barcode_2')
```

Comment:
Oh right! Thank you. 

This might also require some more thinking. I don't think we want to ingest those barcode fields for the mail-in projects considering how much of a headache that was in the beginning (it's the reason why we [removed ingesting the other barcode fields ](https://github.com/seattleflu/id3c-customizations/commit/099a95d67311f795dd5de7aca1ff243f1fecbb05) in the first place).

Reply:
Agreed on not ingesting those for the mail-in projects. That's the approach I took in https://github.com/seattleflu/backoffice/pull/65/commits/9d9809b6c3c0bf5fddbc26130f9e99343485e34f too.

Reply:
Thanks, I'll do similar barcode handling in the ETL.